example/lib/wifi_screen/wifi_bloc.dart

```diff
@@ -40,19 +48,8 @@ class WiFiBlocSoftAP extends Bloc<WifiEvent, WifiState> {
 
 
 
-      var listWifi = null;/* await prov.startScanWiFi() */;
-      /* TODO remove dummy scan result. Scanning is probably impossible on web.
```

Comment:
Scanning is a matter of sending request to ESP AP and receiving response. Since communication is bidirectional. It should be possible regardless of what platform is using this library.

Reply:
I've already restored this code in the later commits (https://github.com/omert08/esp_softap_provisioning/pull/6/commits/e94575954badec04d9f3330df912de70bf16b120). Scanning works fine on Desktop and Web in the last commit here.

I recommend you only review the overall changes, there were a bunch of automatic formatting changes that I cleaned up later.